waterlog/src/Components/DailyCostsReports.js

```diff
@@ -20,24 +22,24 @@ const DailyCostsReports = (props) => {
     labels: labelX,
     datasets: [
       {
-        label: 'cost',
+        label: 'current cost',
         data: dataY,
         fill: true,
         borderColor: '#56ccf7',
```

Comment:
You are using these colours a lot make use of constants, please. Same goes for `#C0C0C0`

Reply:
Thanks its being fixed: https://github.com/Hunter27/WaterLog-Front-end/pull/106